microcosm_pubsub/dispatcher.py

```diff
@@ -110,6 +111,9 @@ def handle_message(self, message, bound_handlers) -> MessageHandlingResult:
                 try:
                     self.validate_message(message)
                     handler = self.find_handler(message, bound_handlers)
+                    self.opaque["X-Request-Handler"] = handler.__name__
+                    # XXX tech debt story to fix setting of metrics name
+                    self.opaque["X-Request-Daemon"] = environ.get("METRICS_NAME", "Unknown")
```

Comment:
This isn't good, but it's a pattern we're grandfathered into from `microcosm-metrics`, here: https://github.com/globality-corp/microcosm-metrics/blob/develop/microcosm_metrics/factories.py#L27

We should go back and fix microcosm metrics to load this information through standard loaders and bind it into the graph to a universally accessibly place so this won't break in environments where we don't use config through environment variables.